src/components/Login.tsx

```diff
@@ -88,7 +89,11 @@ class Component extends React.Component {
     if (response.status === 200) {
       const jwt = response.data;
       const user = jwtDecode(jwt).context.user;
-      await props.effects.setUser(user);
+      // TODO: use id instead of email?
+      const egoId = user.email;
```

Comment:
Yes, if we can use IDs now then let's do that

Reply:
Ids aren't returned in the jwt, so it would have to be a separate call right now. 
Could we add the id to the jwt @ra-ver ?

Reply:
It's being returned at the root level under as "sub"

Reply:
oh, ok! so sub === id?

Reply:
Yup